client/src/components/Address/CreateAddress.js

```diff
@@ -32,7 +32,7 @@ const CreateAddress = () => {
   };
   let submitHandler;
   let id = location.pathname.slice(14);
-  if (location.pathname == "/user/address/null") {
+  if (location.pathname == "/user/address/new") {
```

Comment:
this line seems to be duplicated multiple times within the same file, can we move it to a const or something like that and just call it in all the places?

Reply:
changed